doc/avx_kernels/implementation.rst

```diff
@@ -100,13 +100,13 @@ When such a function is given, we can choose a proper function to call for a giv
 As the wire the gate applies to for the internal function should be a compile-time parameter, we need a switch-case statement for calling internal functions.
 Still, one might see that all single-qubit gate functions share the same structure.
 Thus it might be beneficial to make a simple helper function that automatically finds a target function depending on the given information.
```

Comment:
```suggestion
One might see that all single-qubit gate functions share the same structure.
Thus, it might be beneficial to make a simple helper function that automatically finds a target function depending on the given information.
```

Not a big fan of `thus` in documentation. Maybe something simpler here?

Reply:
Paraphrased, "Since all single-qubit gate functions share the same structure, it might be beneficial to make a simple helper function that automatically finds a target function depending on the given information." Sound good?

Reply:
Perfect.